pandas/core/frame.py

```diff
@@ -2158,6 +2162,9 @@ def to_html(self, buf=None, columns=None, col_space=None, header=True,
             Convert the characters <, >, and & to HTML-safe sequences.
         notebook : {True, False}, default False
             Whether the generated HTML is for IPython Notebook.
+
+            .. deprecated:: 0.25.0
```

Comment:
is there a replacement?

Reply:
there would be no replacement from an end user perspective. see https://github.com/pandas-dev/pandas/pull/26815#issuecomment-501402253